lisa/sut_orchestrator/azure/arm_template.json

```diff
@@ -70,6 +70,34 @@
                 "description": "tags of availability set"
             }
         },
+        "use_existing_virtual_network": {
+            "type": "bool",
+            "defaultValue": false,
+            "metadata": {
+                "description": "flag to use an existing vnet"
+            }
+        },
+        "virtual_network_resource_group": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]",
```

Comment:
Is the default value used?

Reply:
i had hoped it would be, but in practice it didn't.  i'll remove the default values from here.